src/sidebar/store/create-store.js

```diff
@@ -7,6 +7,19 @@ import immutable from '../util/immutable';
 
 import { createReducer, bindSelectors } from './util';
 
+/**
+ * // Base redux store
+ * @typedef {import("redux").Store} ReduxStore
+ *
+ * // Custom stores
+ * @typedef {import("./modules/activity").ActivityStore} ActivityStore
+ * // TODO: add more stores
+ *
+ * // Combine all stores
+ * @typedef {ReduxStore &
+ *  ActivityStore} SidebarStore
```

Comment:
Ah - I've realized that these types should be moved to `store/index.js` because the set of modules that the store has is determined by the `store` function in `store/index.js` whereas `create-store.js` is a generic utility.

Reply:
yes, that makes more sense